src/Toaster/Toast.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { ToastType } from 'react-toastify';
+import { IToast } from '.';
+import { Box, IBoxCSS } from '../Box';
+import { Icon } from '../Icon';
+import { Text } from '../Text';
+
+export const iconMap = {
+  [ToastType.INFO]: 'info-circle',
+  [ToastType.WARNING]: 'exclamation-triangle',
+  [ToastType.ERROR]: 'times-circle',
+  [ToastType.SUCCESS]: 'check-circle',
+  [ToastType.DEFAULT]: 'lightbulb',
+};
+
+export const Toast = React.forwardRef<HTMLElement, IToast>((props, ref) => {
+  const { content, type, css, closeToast, ...rest } = props;
+
+  return (
+    <Box {...rest} ref={ref} css={[toastStyles(), css]}>
+      <Icon icon={iconMap[type]} />
```

Comment:
Shouldn't the icons have some different colors depending on severity?
Would be great to make them at least themable.